### PR TITLE
Add documentation on user error for non-supported keywords

### PR DESCRIPTION
**Pull request by robnewman**
# Description
Add docs on non-supported keywords in JSON schemas for workflows

# TODO

- [x] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

File: docs/advanced-features/workflows.md [hunk 1]
```diff
@@ -423,5 +423,6 @@ and
 
 ## Known limitations
 * Only [Draft 7 Json Schemas](https://json-schema.org/specification-links.html#draft-7) are supported
+  * If a workflow schema includes a non-supported keyword, the user interface displays an `unknown keyword: <non-supported keyword>` error
```
Comment (fiskus) [inline docs/advanced-features/workflows.md:426]: This line is longer than 80 symbols (as line above and beneath)

Comment (fiskus) [inline docs/advanced-features/workflows.md:426]:
```suggestion
  * If a workflow schema includes a non-supported keyword,
    the user interface displays an `unknown keyword: <non-supported keyword>` error
```

**Review by fiskus — APPROVED** [review]

File: docs/advanced-features/workflows.md (hunk 1 above)
Comment (fiskus) [reply docs/advanced-features/workflows.md:426]: I'm not sure my suggestion is valid

Comment (robnewman) [reply docs/advanced-features/workflows.md]: Fixed in latest commit

**Pull request merged**